storage/tianmu/loader/load_parser.h

```diff
@@ -49,32 +49,32 @@ class LoadParser final {
   ~LoadParser() = default;
 
   uint GetPackrow(uint no_of_rows, std::vector<ValueCache> &vcs);
-  int64_t GetNumOfRejectedRows() const { return rejecter.GetNumOfRejectedRows(); }
-  bool ThresholdExceeded(int64_t no_rows) const { return rejecter.ThresholdExceeded(no_rows); }
+  int64_t GetNumOfRejectedRows() const { return rejecter_.GetNumOfRejectedRows(); }
+  bool ThresholdExceeded(int64_t no_rows) const { return rejecter_.ThresholdExceeded(no_rows); }
   int ProcessInsertIndex(std::shared_ptr<index::RCTableIndex> tab, std::vector<ValueCache> &vcs, uint no_rows);
-  int64_t GetNoRow() const { return row_no; }
-  int64_t GetDuprow() const { return dup_no; }
+  int64_t GetNoRow() const { return row_no_; }
+  int64_t GetDuprow() const { return dup_no_; }
 
  private:
-  RCAttrPtrVect_t &attrs;
+  RCAttrPtrVect_t &attrs_;
 
-  std::vector<int64_t> last_pack_size;
-  int64_t start_time;
+  std::vector<int64_t> last_pack_size_;
+  int64_t start_time_ = 0;
 
-  ReadBuffer read_buffer;
+  ReadBuffer read_buffer_;
 
-  std::shared_ptr<ParsingStrategy> strategy;
-  std::shared_ptr<index::RCTableIndex> tab_index;
+  std::shared_ptr<ParsingStrategy> strategy_;
+  std::shared_ptr<index::RCTableIndex> tab_index_;
 
-  const char *cur_ptr;
-  const char *buf_end;
-  const system::IOParameters &ioparam;
-  uint pack_size;
-  Rejecter rejecter;
-  uint cur_row = 0;
-  int64_t no_obj = 0;
-  int64_t row_no = 0;
-  int64_t dup_no = 0;
+  const char *cur_ptr_;
+  const char *buf_end_;
+  const system::IOParameters &ioparam_;
+  uint pack_size_ = 0;
+  Rejecter rejecter_;
+  uint cur_row_ = 0;
+  int64_t no_obj_ = 0;
```

Comment:
Here, using `no_obj` to `num_of_obj_`.

Do you know the meaning of `row_no_` and `dup_no_` ? what are they used for?